charts/rhtap-backing-services/templates/openshift-acs/job-stackrox-api.yaml

```diff
@@ -0,0 +1,58 @@
+{{- if .Values.backingServices.acs.enabled -}}
+  {{- $acs := .Values.backingServices.acs -}}
+  {{- $name := printf "%s-post-deploy-%d" $acs.name .Release.Revision }}
```

Comment:
Why use of the revision #?

Reply:
Let's tackle the revision numbers, and the Helm labels to delete/prune jobs in a another PR, as we've spoken.